python/docs/source/getting_started/installation.rst

```diff
@@ -0,0 +1,119 @@
+..  Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+..    http://www.apache.org/licenses/LICENSE-2.0
+
+..  Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+============
+Installation
+============
+
+The official release channel is to download it from `the Apache Spark website <https://spark.apache.org/downloads.html>`_.
+Alternatively, you can also install it via pip from PyPI.  PyPI installation is usually to use
+standalone locally or as a client to connect to a cluster instead of setting a cluster up.  
+ 
+This page includes the instructions for installing PySpark by using pip, Conda, downloading manually, and building it from the source.
+
+Python Version Supported
+------------------------
+
+Python 3.6 and above.
+
+Using PyPI
+----------
+
+PySpark installation using `PyPI <https://pypi.org/project/pyspark/>`_
+
+.. code-block:: bash
+
+    pip install pyspark
+	
+Using Conda  
+-----------
+
+Conda is an open-source package management and environment management system which is a part of `Anaconda <https://docs.continuum.io/anaconda/>`_ distribution. It is both cross-platform and language agnostic.
+  
+Conda can be used to create a virtual environment from terminal as shown below:
+
+.. code-block:: bash
+
+	conda create -n pyspark_env 
+
+After the virtual environment is created, it should be visible under the list of conda environments which can be seen using the following command:
+
+.. code-block:: bash
+
+	conda env list
+
+The newly created environment can be accessed using the following command:
+
+.. code-block:: bash
+
+	conda activate pyspark_env
+
+In lower Conda version, the following command might be used:
```

Comment:
In earlier Conda versions ... should be used:
(earlier than what?)

Reply:
Hi @srowen,

Based on the conda documentation the answer is "Conda version before 4.4". 
[The Source for this info is ](https://docs.conda.io/projects/conda/en/latest/release-notes.html#id208)

> conda **activate:** The logic and mechanisms underlying environment activation have been reworked. With conda 4.4, conda activate and conda deactivate are now the preferred commands for activating and deactivating environments. You'll find they are much more snappy than the source activate and source deactivate commands from previous conda versions. The conda activate command also has advantages of (1) being universal across all OSes, shells, and platforms, and (2) not having path collisions with scripts from other packages like Python virtualenv's activate script.

Reply:
Right, I'm suggesting saying that here.